app/src/main/java/nextstep/payments/ui/screen/NewCardScreen.kt

```
@@ -231,6 +285,106 @@ private fun PasswordInputField(
     )
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun BankBottomModalSheet(
+    sheetState: SheetState,
+    onBankClick: (BankType) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    ModalBottomSheet(
+        modifier = modifier.semantics {
+            contentDescription = "bankBottomSheet"
+        },
+        sheetState = sheetState,
+        onDismissRequest = {},
+    ) {
+        Column(
+            modifier = modifier,
+            verticalArrangement = Arrangement.spacedBy(8.dp),
+        ) {
+            // 은행 리스트
+            BankSelectRow(
+                onBankClick = { bankType ->
+                    onBankClick(bankType)
+                },
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+private fun BankSelectRow(
+    onBankClick: (BankType) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val bankList = BankType.getBankList()
+
+    FlowRow(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(horizontal = 45.dp, vertical = 35.dp),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalArrangement = Arrangement.spacedBy(23.dp),
+        maxItemsInEachRow = 4
+    ) {
+        bankList.forEach { bankType ->
+            BankItem(
+                bankName = stringResource(bankType.bankNameResId),
+                bankImage = painterResource(bankType.bankImageRes!!),
```

여기서 !!를 사용하지 않으려면 어떻게 하는게 좋을까요?
enum에 NOT_SELECTED이 정말 필요할지 고민해보는것도 좋을 것 같습니다.

---

현재 NOT_SELECTED 을 활용하여 구성한 로직이 많아 NOT_SELECTED 제거한다면 전체 로직 설계를 재구성해야 하는 상황입니다.일단 if 문을 활용하여 !!를 사용하지 않도록 구성해보았습니다만 애초부터 잘못된 설계였을가요...?

---

현재 BankType.NOT_SELECTED 활용하여 초기 Payment의 Default 값을 설정하고 NewCardScreen LaunchedEffect에서 sheetState의 show, hide 로직을 관리하고 있어 if 문을 활용해 !! 를 지우도록 처리해보았습니다.
물론 위 sheetState의 show, hide 로직은 다른 방법으로도 얼마든지 구현할 수 있지만 초기 상태를 둠으로써 이용자에게 선택하지 않은 상태를 보여주기 위한 의도로 구현해보았는데 혹시 잘못된 방법일가요..?

---

저는 BankType에 NOT_SELECTED 대신 null을 이용하면 되지 않을까 라는 생각을 했습니다.
null을 사용하는 것과 NOT_SELECTED를 추가해서 사용하는 것과 어떤 것이 더 좋은 선택일까요?

지금은 카드사를 선택하는 기능만 있어서 그렇지만 BankType을 선택하냐 안하냐로만 사용하지 않고 다른 것으로 사용하게 될 일은 없을까요?